cupy/cuda/cupy_cudnn.h

```diff
@@ -293,12 +317,38 @@ cudnnStatus_t cudnnBatchNormalizationBackward(...) {
     return CUDNN_STATUS_NOT_SUPPORTED;
 }
 
+cudnnStatus_t cudnnSetPooling2dDescriptor_v4(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
+cudnnStatus_t cudnnSetPoolingNdDescriptor_v4(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
+cudnnStatus_t cudnnCreateActivationDescriptor(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
+cudnnStatus_t cudnnSetActivationDescriptor(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
+cudnnStatus_t cudnnDestroyActivationDescriptor(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
+cudnnStatus_t cudnnActivationForward_v4(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
+cudnnStatus_t cudnnActivationBackward_v4(...) {
+    return CUDNN_STATUS_NOT_SUPPORTED;
+}
+
 #endif // #if defined(CUPY_NO_CUDA) || (CUDNN_VERSION < 4000)
 
 
-#if defined(CUPY_NO_CUDA) || (CUDNN_VERSION < 5000)
```

Comment:
Because I rename functions of this section.
For example `cudnnSetPooling2dDescriptor_v4`.